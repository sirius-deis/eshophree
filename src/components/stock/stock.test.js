import {render, screen} from '@testing-library/react'
import Stock from './stock'

describe('Stock component', () => {
  it('should match snapshot', () => {
    const {container} = render(<Stock total={10} sold={3} endIn={Date.now() + 1000000} />)
    expect(container).toMatchSnapshot()
  })
  it('should render values correctly', () => {
    render(<Stock total={10} sold={3} endIn={Date.now() + 1000000} />)
    expect(screen.getByText('Available: 7')).toBeInTheDocument()
    expect(screen.getByText('Sold: 3')).toBeInTheDocument()
  })
})