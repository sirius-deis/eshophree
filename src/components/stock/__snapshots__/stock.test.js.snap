// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stock component should match snapshot 1`] = `
<div>
  <div
    class="sc-bgqQcB beiWqX"
  >
    <div
      style="margin-bottom: 1rem;"
    >
      <div
        class="sc-ewnqHT dLOUny"
      >
        <span
          style="font-size: 1.3rem;"
        >
          Available: 
          7
        </span>
        <span
          style="font-size: 1.3rem;"
        >
          Sold: 
          3
        </span>
      </div>
      <div
        class="progress"
      >
        <div
          class="sc-gTRrQi gjhVB"
          width="100"
        />
        <div
          class="sc-gTRrQi kFAbuf"
          width="70"
        />
      </div>
    </div>
    <div
      class="sc-ewnqHT fcqwuT"
    >
      <div
        style="display: flex; flex-direction: column; gap: 0.5rem;"
      >
        <h4
          style="font-size: 1.7rem;"
        >
          Harry Up
        </h4>
        <p
          style="font-size: 1.1rem;"
        >
          offer end in:
        </p>
      </div>
      <div
        class="sc-ewnqHT jRSYnd"
      >
        <div
          class="clock"
        >
          <span
            class="circle"
          >
            00
          </span>
          Days
        </div>
         
        :
        <div
          class="clock"
        >
          <span
            class="circle"
          >
            00
          </span>
          Hours
        </div>
         
        :
        <div
          class="clock"
        >
          <span
            class="circle"
          >
            16
          </span>
          Mins
        </div>
         
        :
        <div
          class="clock"
        >
          <span
            class="circle"
          >
            39
          </span>
          Secs
        </div>
      </div>
    </div>
  </div>
</div>
`;
